modelcontextprotocol/tools/assets.py

```diff
@@ -60,24 +62,77 @@ def update_assets(
 
         # Create assets with updated values
         assets = []
-        index = 0
-        for updatable_asset in updatable_assets:
+        # readme_update_parent_assets: Assets that were updated with readme.
+        readme_update_parent_assets = []
+        for index, updatable_asset in enumerate(updatable_assets):
             type_name = updatable_asset.type_name
+            qualified_name = updatable_asset.qualified_name
             asset_cls = getattr(
                 __import__("pyatlan.model.assets", fromlist=[type_name]), type_name
             )
             asset = asset_cls.updater(
                 qualified_name=updatable_asset.qualified_name,
                 name=updatable_asset.name,
             )
-            setattr(asset, attribute_name.value, attribute_values[index])
-            assets.append(asset)
-            index += 1
-        response = client.asset.save(assets)
 
-        # Process response
+            # Special handling for README updates
+            if attribute_name == UpdatableAttribute.README:
+                # Get the current readme content for the asset
+                # The below query is used to get the asset based on the qualified name and include the readme content.
+                asset_readme_response = (
+                    FluentSearch()
+                    .select()
+                    .where(CompoundQuery.asset_type(asset_cls))
+                    .where(asset_cls.QUALIFIED_NAME.eq(qualified_name))
+                    .include_on_results(asset_cls.README)
+                    .include_on_relations(Readme.DESCRIPTION)
+                    .execute(client=client)
+                )
+
+                if first := asset_readme_response.current_page():
+                    try:
+                        current_content = first[0].readme.description
+                        logger.info(
+                            f"Current readme content for {qualified_name}: {current_content}"
+                        )
+                    except Exception as e:
+                        logger.error(f"Error getting current readme content: {e}")
```

Comment:
Is this an error?

Reply:
Resolved via : [e5f1291](https://github.com/atlanhq/agent-toolkit/pull/63/commits/e5f12913c4d119fbb37a300690da140528fef95a)